web/js/signup_comps/Auth.js

```diff
@@ -68,16 +83,100 @@ var SocialAuth = React.createClass({
     }
 });
 
+var wasClicked = {
+  "Google": null,
+  "Facebook": null,
+  "Twitter": null,
+  "Tumblr": null,
+  "Reddit": null,
+  "Instagram": null,
+  "Spotify": null
+};
+var sharePoints = 0;
+var clickBtns = 0;
```

Comment:
`clickBtns` and `sharePoints` don't seem to be used (nor should they be since `wasClicked` has all the info we need)